addon/utils/dom.js

```diff
@@ -76,7 +76,10 @@ function closest(el, selector) {
 
 export function getDestinationElement(context) {
   let dom = getDOM(context);
-  let destinationElement = findElementById(dom, 'ember-bootstrap-wormhole');
+  const id = 'ember-bootstrap-wormhole';
+  console.log('dom', dom);
+  let destinationElement = findElementById(dom, id) || findElemementByIdInShadowDom(context, id);
+  // AFJLAJLSFJ AFJ LJ
```

Comment:
`console.log()` and the comment looks like left-over from debugging.

Reply:
Removed.